pkg/process/checks/net.go

```diff
@@ -208,6 +209,14 @@ func batchConnections(
 			EncodedDNS:        dnsEncoder.Encode(batchDNS),
 			ContainerHostType: cfg.ContainerHostType,
 		}
+
+		// Add OS telemetry
+		if hostInfo := host.GetStatusInformation(); host != nil {
```

Comment:
Should it be a check for `hostInfo != nil`?

Reply:
Thanks!